tools/gyp/pylib/gyp/generator/make.py

```diff
@@ -944,7 +948,14 @@ def WriteActions(self, actions, extra_sources, extra_outputs,
       # libraries, but until everything is made cross-compile safe, also use
       # target libraries.
       # TODO(piman): when everything is cross-compile safe, remove lib.target
-      self.WriteLn('cmd_%s = LD_LIBRARY_PATH=$(builddir)/lib.host:'
+      if self.flavor == 'zos':
+        self.WriteLn('cmd_%s = LIBPATH=$(builddir)/lib.host:'
```

Comment:
May want to adjust the indents in all added `if`s below so the alignment matches that before the change.